src/admin.rs

```diff
@@ -68,3 +68,99 @@ impl From<NativeFluvioAdmin> for FluvioAdmin {
         }
     }
 }
+
+#[cfg(feature = "unstable")]
```

Comment:
I think that we need to add [cargoArgs](https://github.com/wasm-tool/rollup-plugin-rust#build-options) to the rollup config, to enable this feature

Reply:
I have tried loading this into cloud and it was successfully using the code in "unstable" so I think we're okay?